full_script.py

```diff
@@ -0,0 +1,294 @@
+"""
+    Alex Chapman
+    2/19/17
+
+    Software Design Project 3
+
+    Finding a list of U.S. Representatives and analyzing the sentiment of
+    their respective wikipedia pages in corrolation with party affiliation.
+"""
+
+import requests
+import pickle
+import wikipedia
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+import matplotlib.pyplot as plt
+
+sent = SentimentIntensityAnalyzer()
+rep_normalized = 0
+dem_normalized = 0
+
+
+class Point(object):
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+
```

Comment:
Hmm, not sure how this class fits into the rest of program. A docstring comment or some kind of documentation to describe what this does would be helpful.